components/action-mgt/org.wso2.carbon.identity.action.management/pom.xml

```diff
@@ -151,7 +159,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.50</minimum>
+                                            <minimum>0.80</minimum>
                                         </limit>
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
```

Comment:
Can we have the complexity back at 0.6 ?
Does that breaks the build ? If so why ?

Reply:
This is updated with 4cd2b36e177012dc1fb1909deac8b37495c62a7c

Reply:
Ack will update that

Reply:
This is updated with 3aa387bca6568151ba382320d0de9a46686c6d97